apps/dcos_l4lb/testdata/state6.json

```diff
@@ -0,0 +1,249 @@
+{
```

Comment:
looks like this is a handcrafted state.json.  Can we take a real state.json instead?